pydra/engine/core.py

```diff
@@ -457,7 +457,7 @@ def __init__(
             messenger_args=messenger_args,
         )
 
-        self.graph = nx.DiGraph()
+        self.graph = Graph()
```

Comment:
can we make the class networkx compatible?

Reply:
and by this you mean?

Reply:
`DiGraph()`